draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -914,11 +914,12 @@ Example use cases include:
    Wi-Fi will be used and Cellular only if the Wi-Fi path is congested or not
    available.
 2) Setting Wi-Fi path to Primary and Cellular to Standby. In this case Wi-Fi
-   will be used and Cellular only if the Wi-Fi is not available. Such setting 
-   results in using the Cellular path only temporally, if more capacity is 
-   needed than the WiFi path can provide.
+   will be used and Cellular only, if the Wi-Fi path is not available or congested. 
+   Such setting results in using the Cellular path only temporally, if more 
+   capacity is needed than the WiFi path can provide, indicating a clear priority
+   of the Wi-Fi path over the Cellular due to e.g. cost reasons.
```

Comment:
This confuses me. What is no the difference between the here explained Standby path and the Secondary path in the above Example 1. In both cases it says,  that the non-Primary path can be used when the Primary path is congested or not available.

My understanding would be that Example 1 - Primary/Secondary - facilitates simultaneous usage (aggregation/splitting) and Primary/Standby is single-path usage with the ability for seamless handover.